.gitignore

```diff
@@ -24,6 +24,7 @@ gravitee.*.log.*/*.log
 gravitee-**-webui/node/
 **/.settings
 **/report.*.json
+package-lock.json
```

Comment:
We don't need to include this in the `.gitignore` file because we do not use `npm` anymore in the project. So it should be fine to reverse this change 👍

Reply:
Done